cypress/integration/journalistCanCreateArticle.feature.js

```diff
@@ -1,31 +1,8 @@
 describe("Journalist can create an article", () => {
   beforeEach(() => {
-    cy.server();
-    cy.visit("/");
-    cy.route({
-      method: "POST",
-      url: "http://localhost:3000/api/auth/*",
-      response: "fixture:successful_login.json",
-      headers: {
-        uid: "user@mail.com",
-      },
-    });
-    cy.route({
-      method: "GET",
-      url: "http://localhost:3000/api/auth/*",
-      response: "fixture:successful_login.json",
-      headers: {
-        uid: "user@mail.com",
-      },
-    });
-    cy.get("#login-form").within(() => {
-      cy.get("#email").type("user@mail.com");
-      cy.get("#password").type("password");
-      cy.get("Button").contains("Submit").click();
-    });
+    cy.login('journalist');
```

Comment:
This may seem out of scope, but was necessary as the fixture this test was using was no longer working with implemented code. We refactored into our newly created login helper, instead of changing the fixtures in the old code.